lib/ui/pages/account_dashboard.dart

```diff
@@ -14,116 +14,88 @@ import 'package:pispapp/ui/widgets/transaction_tile.dart';
 import '../theme/light_theme.dart';
 
 class AccountDashboard extends StatelessWidget {
-  void _showTransactionDetailBottomSheet(Transaction t) {
-    Get.bottomSheet<dynamic>(
-      TransactionBottomSheet(t),
+  @override
+  Widget build(BuildContext context) {
+    return GetBuilder<AccountDashboardController>(builder: (controller) {
+      if (controller.isLoading) {
+        return _buildLoadingWidget();
+      } else {
+        return SafeArea(
+          child: RefreshIndicator(
+            onRefresh: () async {
+              Get.find<AccountDashboardController>().refresh();
+            },
+            child: SingleChildScrollView(
```

Comment:
Did you say there was a scrolling problem here? I can try to fix it after this is merged.

Reply:
not, this one, i think its in the `payment_confirmation.dart` file

Reply:
Ok